packages/nvidia_nat_memmachine/tests/test_memmachine_api_calls.py

```
@@ -0,0 +1,619 @@
+        api_spy: APICallSpy,
+        spied_memory_instance: Mock
+    ):
+        """Verify that search calls memory.search() with correct parameters."""
+        # Set up search return value
+        spied_memory_instance.search.return_value = {
+            "episodic_memory": [{"content": "I like pizza", "metadata": {}}],
+            "semantic_memory": [],
+            "episode_summary": []
+        }
+        
+        results = await editor_with_spy.search(
```

Ruff (0.14.13)

Local variable `results` is assigned to but never used

Remove assignment to unused variable `results`

(F841)